src/main/pg/timerio.c

```diff
@@ -29,6 +29,13 @@
 
 #include "timerio.h"
 
+/*
+    idx      => is the index in the config array,
+    pin      => is the pin to be mapped,
+    pos      => is the pin position (index) within timerHardware (full) that should be used
+                e.g. 1 = first occurence of the pin, 2 = second occurence, etc.
```

Comment:
can we rename 'pos' to 'pinOccurenceInTimerMap' and make it zero based?

`pos` is bad.  the word `occurence` was the key to unlocking understanding of how it works and this should be reflected in the choice of argument naming.

Reply:
Have updated the comment to document it, and shortened the naming given it is a macro - and code completion won't be available anyway.

I don't think adjusting the index is a good idea, as it will be a breaking change for existing configs as stored and has been 1 based since timerIOConfig was implemented.